Gopkg.lock

```diff
@@ -34,24 +34,7 @@
 [[projects]]
   branch = "master"
   name = "github.com/docker/distribution"
-  packages = [
```

Comment:
Please upgrade your local `dep` to the latest version :)

Reply:
Thanks for reminding. 

I have done (go get -u github.com/golang/dep/cmd/dep) but no change after doing dep ensure!. Will look into it.

Reply:
Running dep version gives:

dep:
 version     : devel
 build date  : 
 git hash    : 
 go version  : go1.8.3
 go compiler : gc
 platform    : linux/amd64
 features    : ImportDuringSolve=false

Reply:
Looking at git commits in the `src/github.com/golang/dep/cmd/dep` dir (master branch) shows:

```commit f9cb443b21244c2c2b8f9281014936300793eea6
Merge: 342e0b7 f217a9c
Author: Carolyn Van Slyck <me@carolynvanslyck.com>
Date:   Tue Apr 24 09:20:03 2018 -0500

    Merge pull request #1779 from wknapik/patch-1
    
    Updated Arch installation instructions.
```

Reply:
All sorted in the end though lots of magic going on. Thanks for helping out Aaron.